File: Data Structures & Algorithms/is-palindrome/submission-3.go
func isPalindrome(s string) bool {
	if len(s) == 0 {
		return false
	}
	
	clean := ""
	for _, ch := range strings.ToLower(s) {
		if (ch >= 'a' && ch <= 'z') || (ch >= '0' && ch <= '9') {
    		clean += string(ch)
		}
	}

	left, right := 0, len(clean)-1
	for left < right {
		if clean[left] != clean[right] {
			return false
		}
		left++
		right--
	}
	return true
}

File: Data Structures & Algorithms/level-order-traversal-of-binary-tree/submission-3.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution:
    def levelOrder(self, root: Optional[TreeNode]) -> List[List[int]]:
        result = []

        if root is None:
            return result

        def bfs(root):
            queue = deque([root])

            while len(queue) > 0:
                qLen = len(queue)
                lvlValues = []

                for i in range(qLen):
                    node = queue.popleft()
                    lvlValues.append(node.val)

                    if node.left is not None:
                        queue.append(node.left)

                    if node.right is not None:
                        queue.append(node.right)

                result.append(lvlValues)

        bfs(root)
        return result